backend/tests/unit/mockdal/Pims.Dal.Mock.Test.csproj

```diff
@@ -0,0 +1,38 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
New test library - Moq.EntityFrameworkCore required to allow testing of keyless entities.
Is not compatible with in memory test database.
Created a new testing library to isolate tests that will use the above library.